dot/rpc/websocket_integration_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build integration
```

Comment:
subscription/websocket_integration_test.go also needs this header I believe

Reply:
Indeed :rocket: Fixed!